### DISCUSSION TITLE
Is there an overview of twitter ratios?

### DISCUSSION THREAD

**Discussion by OnlyOneMoreHour** [tweepy/tweepy#1455] [Questions] [answered]
Hello

My account got ghost banned after I used tweepy. Is there a overview of ratios for established and new accounts in twitter?

**Comment by Harmon758** [comment, marked as answer]
I'm not sure what you mean by ratios.
If you mean rate limits, see https://developer.twitter.com/en/docs/twitter-api/v1/rate-limits.
